README.aspnet.md

```diff
@@ -0,0 +1,103 @@
+﻿# Featured Tags
+
+* `4.8`
+  * `docker pull mcr.microsoft.com/dotnet/framework/aspnet:4.8`
+* `3.5`
+  * `docker pull mcr.microsoft.com/dotnet/framework/aspnet:3.5`
+
+# About This Image
+
+ASP.NET is a high productivity framework for building Web Applications using Web Forms, MVC, Web API and SignalR.
+
+This repository contains `Dockerfile` definitions for ASP.NET Docker images. 
+
+This image contains:
+- Windows Server Core as the base OS
+- IIS 10 as Web Server
+- .NET Framework (multiple versions available)
+- .NET Extensibility for IIS
+
+# How to use these Images
+
+The [.NET Framework Docker samples](https://github.com/microsoft/dotnet-framework-docker/blob/master/samples/README.md) show various ways to use .NET Framework and Docker together.
+
+## Container sample: Run an ASP.NET application
+You can quickly run a container with a pre-built [sample ASP.NET Docker image](https://hub.docker.com/_/microsoft-dotnet-framework-samples/), based on the [ASP.NET Docker sample].
+
+Type the following [Docker](https://www.docker.com/products/docker) command:
+
+```console
+docker run --name aspnet_sample --rm -it -p 8000:80 mcr.microsoft.com/dotnet/framework/samples:aspnetapp
+```
+
+After the application starts, navigate to `http://localhost:8000` in your web browser. You need to navigate to the application via IP address instead of `localhost` for earlier Windows versions, which is demonstrated in [View the ASP.NET app in a running container on Windows](https://github.com/microsoft/dotnet-framework-docker/blob/master/samples/aspnetapp/README.md#view-the-aspnet-app-in-a-running-container-on-windows).
+
+For a comprehensive tutorial on running an ASP.NET app in a container, check out [the tutorial on the docs site](https://docs.microsoft.com/en-us/dotnet/articles/framework/docker/aspnetmvc).
```

Comment:
Should this reference be moved to the ASP.NET samples readme?  Outside of this container sample, we try to centralize all content within the samples readme.